gubernator/test-gubernator.sh

```diff
@@ -18,8 +18,7 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
```

Comment:
add `cd "$(dirname "$0")"` here, since I don't think bootstrap will set the working dir to gubernator/.